src/Crocodealer/Config.hs

```diff
@@ -1,18 +1,40 @@
+{- | Configurations through the @config.toml@ file@.
+-}
+
 module Crocodealer.Config
        ( Repo (..)
        , Config (..)
+       , loadConfig
        ) where
 
-import Crocodealer.Core.Label as Label
+import Toml (TomlCodec, (.=))
+import qualified Toml
 
+import Crocodealer.Core.Label as Label
 
 newtype Repo = Repo
     { unRepo :: Text
-    }
+    } deriving (Show)
+
+-- | TOML Codec for a list of 'Repo'
+repoCodec :: Toml.Key -> TomlCodec [Repo]
+repoCodec key = Toml.diwrap (Toml.arrayOf Toml._Text key)
```

Comment:
Not sure if this is the best implementation, it seems to fail to find the `ignoredRepositories` key if it is below the `labelRule` in the config file. Not sure why or if that should happen.

Reply:
The implementation is fine! If you put `ignoredRepositories` after `labelRule` they become a part of `labelRule` table because unlike Haskell, TOML is not layout-sensitive, unfortunately :disappointed: